### PR TITLE
updated plugins documentation

### PR DESCRIPTION
**Pull request by scottinet**
Updated the plugins documentation to match the new plugins behavior, after pull request https://github.com/kuzzleio/kuzzle/pull/253 is merged into Kuzzle master branch

### CODE REVIEW

**Comment by benoitvidis** [comment]
+1

**Comment by dbengsch** [comment]
Wouldn't it be better to warn about the necessity of restarting kuzzle ?
npm is a trademark and if it could lead to rename the config parameter `npmVersion` afterward, it might not worth it.
Apart from my remarks: +1

**Comment by scottinet** [comment]
Thanks @dbengsch, I took account of your remarks and modified the documentation.

File: source/includes/core_documentation/_02-plugins.md [hunk 1]
```diff
@@ -8,36 +8,170 @@ There are several types of plugins:
...
+Options:
+
+  -h, --help                  output usage information
+  --install                   *If plugin [name] is provided, install it using --npmVersion, --gitUrl or --path. Otherwise, (re-)install all listed plugins
+  --get                       *Gets the plugin [name] current stored configuration
+  --set <JSONObject>          *Updates the plugin configuration with new properties
+  --replace <JSONObject>      *Replaces a plugin configuration with a new one
+  --unset <property>          *Deletes the property [property] from the plugin configuration
+  --remove                    *Remove the supplied plugin [name] from Kuzzle
+  --activate                  *Mark the plugin as "activated" (Kuzzle ignores deactivated plugins)
```
Comment (xbill82) [inline source/includes/core_documentation/_02-plugins.md]: there is inconsistency here between 3rd person and 2nd person: sometimes it's "Replace*s*", sometime is "Mark"

**Comment by xbill82** [comment]
A part from my remarks, +1 :)

File: source/includes/core_documentation/_02-plugins.md (hunk 1 above)
Comment (scottinet) [inline source/includes/core_documentation/_02-plugins.md]: Fixed, both in this documentation and in Kuzzle core project.

**Comment by ballinette** [comment]
:+1:

**Pull request merged**